gammapy/maps/wcsnd.py

```diff
@@ -2,8 +2,11 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 import copy
 import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
Matplotlib is actually an optional dependency, this import should go back to the `WcsNDMap.plot()` method.